datahub-web-react/src/app/search/sidebar/PlatformNode.tsx

```diff
@@ -55,15 +55,21 @@ const PlatformNode = () => {
     );
 
     const { isOpen, isClosing, toggle } = useToggle({
-        initialValue: isSelected,
+        initialValue: isPlatformSelectedPrefix,
         closeDelay: 250,
         onToggle: (isNowOpen: boolean) => trackToggleNodeEvent(isNowOpen, 'platform'),
     });
 
-    const onClickHeader = () => {
+    const onClickTriangle = () => {
         if (count) toggle();
     };
 
+    const onClickHeader = () => {
+        const isNowSelected = !isPlatformSelectedExact;
```

Comment:
this variable name is hard to follow... isNowSelected is the opposite of whether the platform alone is selected?

Reply:
Eg. is _what_ now selected?